libraries/src/test/java/com/baeldung/caffeine/CaffeineUnitTest.java

```diff
@@ -0,0 +1,156 @@
+package com.baeldung.caffeine;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import javax.annotation.Nonnull;
+
+import org.junit.Test;
+
+import com.github.benmanes.caffeine.cache.AsyncLoadingCache;
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import com.github.benmanes.caffeine.cache.Expiry;
+import com.github.benmanes.caffeine.cache.LoadingCache;
+
+public class CaffeineUnitTest {
+
+    @Test
+    public void given_Cache_whenPopulate_thenValueStored() {
```

Comment:
Several of the test names begin with "given_".
Let's remove the trailing underscore. So this one, for example, would just be:
"givenCache_whenPopulate_thenValueStored"

Reply:
Done